mysql-schema/mysql-schema-tidb-mdl-view.md

```diff
@@ -0,0 +1,29 @@
+---
+title: mysql.tidb_mdl_view
+summary: Learn about the `tidb_mdl_view` table in the `mysql` schema.
+---
+
+# `mysql.tidb_mdl_view`
+
+This table shows the information about the [metadata lock](/metadata-lock.md) views.
+
+```sql
+DESC mysql.tidb_mdl_view;
+```
+
+The output is as follows:
+
+```
++-------------+-----------------+------+------+---------+-------+
+| Field       | Type            | Null | Key  | Default | Extra |
++-------------+-----------------+------+------+---------+-------+
+| job_id      | bigint          | NO   | PRI  | NULL    |       |
+| db_name     | longtext        | YES  |      | NULL    |       |
+| table_name  | longtext        | YES  |      | NULL    |       |
+| query       | longtext        | YES  |      | NULL    |       |
```

Comment:
Should we clarify the different fields as well?
Looking at https://docs.pingcap.com/tidb/stable/metadata-lock/#observability the query is the DDL, and the session ID and SQL_DIGESTS are from the blocking DML queries.

Also there is a mismatch between the lower case names in this table and the UPPER CASE field names in https://docs.pingcap.com/tidb/stable/metadata-lock/#observability.

Reply:
I've updated the other doc as well to match the v8.5.1 behavior.